Add Player info Element to DELTA4 And DELTA3

Iceotaku opened the pull request:
This Update has been list for #44 And Apply with Next Lazer release 
This update has already created and Test with Lazer PR And i think it's useful to apply with DELTA4 And DELTA 3
Preview
![osu!_YqYgFBShom](https://user-images.githubusercontent.com/68460824/227561629-be3032e5-13da-4d81-947a-a937b517d492.jpg)
![osu!_TURANJJAiA](https://user-images.githubusercontent.com/68460824/227561650-c6c7c396-b46e-4d59-a660-20dd1f80b4b0.jpg)

Darksoz123 commented:
Hi, I noticed that you made some changes to the skin, but I don't know from where to download the updated skin.

Thx in advance. :D

Iceotaku commented:
@Darksoz123 This element i added with Skin editor in Osu!lazer PR Build and Updated to next Osu!lazer release 
![HufblPzwBN](https://user-images.githubusercontent.com/68460824/227752005-5ab275cb-4513-4dd7-bac7-e4bd465afe96.png)
but i have a plan to change this skin project to open source in the future (i guess 🤔🤔🤔🤔)
